enhancements/console/customize-catalogs.md

```diff
@@ -0,0 +1,320 @@
+templates on the dev console but they can't change the default devfiles currently.
+
+As a cluster admin, I want to hide Developer catalog for all users.
+For example for customers who has their own way to 
+install workloads, services, etc. without the developer catalog.
+
+#### Story 2
+
+Customer wants to display only their own customized and validated 
+templates on the dev console but they can't change the default devfiles currently.
+
+As a cluster admin, I want to hide certain sub-catalogs (types) for all users.
+For example for customers who don't want to use Helm charts, 
+or another sub-catalog without disabling Helm support in the whole developer console.
+
+#### Story 3
+
+Customer wants to display only their own customized and validated 
+templates on the dev console.
+
+As a cluster admin, I want to enable certain sub-catalogs (types) for all users.
+
+### Goals
+
+- Allow cluster admins to hide the developer catalog or certain sub-catalogs (types).
+- Allow cluster admins to enable certain sub-catalogs (types) for users.
+- Provide information to admins about sub-catalogs (types) they can hide from users.
+
+### Non-Goals
+
+- Allow cluster admins to customize categories or hide individual items.
+
+## Proposal
+
+We can extend the existing `operator.openshift.io/v1` / `Console CRD`. 
+We can add `spec.customization.developerCatalog.types.state` suggests types are `Enabled` or `Disabled`,
+`spec.customization.developerCatalog.types.disabled` which lists types to hide and 
+`spec.customization.developerCatalog.types.enabled` which lists types to show.
+
+### Workflow Description
+
+To enable the types:
+
+```text
+├── spec
+│   ├── customization
+|       ├── developerCatalog
+|           ├── types
+|               ├── state
+|               ├── enabled
+|                   └──
+└── ...
+```
+
+To disable the types:
+
+```text
+├── spec
+│   ├── customization
+|       ├── developerCatalog
+|           ├── types
+|               ├── state
+|               ├── disabled
+|                   └──
+└── ...
+```
+
+Each catalog conforms to the following schema:
+
+// +optional
+- `types: object` , defines the types to enable or disable.
+
+// +optional
+- `state: Enabled|Disabled` , union discriminator suggests types are `Enabled` or `Disabled`. Default value is `Enabled`.
+
+// +optional
+- `disabled: string[]` , defines a list of types to hide. 
+
+// +optional
+- `enabled: string[]` , defines a list of types to enable. 
+
+If the `state` is `Enabled` and `enabled` list is empty, all the types will be shown.
+If the `state` id `Enabled` and `enabled` list is non-empty, only the `enabled` list items will be shown.
+If the `state` is `Disabled` and `disabled` list is empty, all the types will be hidden.
+If the `state` id `Disabled` and `disabled` list is non-empty, only the `disabled` list items will be hidden.
+
+A type should not be listed both in `enabled` and `disabled` lists.
```

Comment:
They can't be anymore as the two lists are mutually exclusive, can delete this or update it

Reply:
Deleted it.